pages/_type/_id.vue

```diff
@@ -109,10 +109,9 @@
             <span class="value">{{ $dayjs(project.updated).fromNow() }}</span>
           </div>
         </div>
-        <hr v-if="$auth.user" class="card-divider" />
+        <hr class="card-divider" />
```

Comment:
Could this be done on version pages too?

Reply:
Oops, I missed that. Fixed it.